db/schema.rb

```diff
@@ -432,6 +432,7 @@
     t.boolean  "student_report_enabled",                 :default => true
     t.text     "last_report_service_hash"
     t.boolean  "show_submit_button",                     :default => true
+    t.boolean  "activity_player_only"
```

Comment:
Is this boolean supposed to be here? This PR doesn't seem to have a migration which adds it.

Reply:
That snuck in from a different branch. I've removed it.